src/graphql/resolvers.ts

```diff
@@ -153,6 +153,18 @@ const resolvers = {
 
     boltsCount: (node: ClimbGQLQueryType) => node.boltsCount ?? -1,
 
+    pitches: (node: ClimbGQLQueryType) => node.pitches != null
+      ? node.pitches.map(pitch => {
+        const { parentId, ...otherPitchProps } = pitch
+        return {
+          id: pitch._id?.toUUID().toString(),
+          uuid: pitch._id?.toUUID().toString(),
```

Comment:
what do you think if we just expose `id` and remove `uuid`? (since they both serve the same purpose)

Reply:
Sure. I was copying it from https://github.com/l4u532/openbeta-graphql/blob/4fd8509f79fd59c04cc39430a2c01ad2d0f8dfe2/src/graphql/resolvers.ts#L134C1, I assumed it might have a specific motivation to expose both.